admin/new-relic-admin.php

```diff
@@ -94,37 +106,46 @@
 					<input class="button-primary" type="submit" value="Submit" name="rtp-relic-get-browser-submit">
```

Comment:
Make value `Submit` string translatable using __()

Reply:
i solved this issue. please review changes.